ios/RNMagicScript/components/UiNodes/UiDropdownList/UiDropdownListNode.swift

```diff
@@ -162,7 +126,11 @@ import SceneKit
 
     @discardableResult
     @objc override func addChild(_ child: TransformNode) -> Bool {
-        guard let dropDownListItem = child as? UiDropdownListItemNode else { return false }
+        guard let dropDownListItem = child as? UiDropdownListItemNode else {
+            return super.addChild(child)
```

Comment:
Is it safe? Looks like this line allows to `inject` other nodes into DropdownList.

Reply:
This is safe and intended. I think we should allow adding any child-component to any parent-component and leave the decision (whether it makes sense) to the developer 🤔

Reply:
👍